gcloud/bigquery/client.py

```diff
@@ -133,3 +134,22 @@ def copy_table(self, name, destination, *sources):
         :returns: a new ``CopyJob`` instance
         """
         return CopyJob(name, destination, sources, client=self)
+
+    def extract_table_to_storage(self, name, source, *destination_uris):
+        """Construct a job for copying one or more tables into another table.
+
+        :type name: string
+        :param name: Name of the job.
+
+        :type source: :class:`gcloud.bigquery.table.Table`
+        :param source: table to be extracted.
+
+        :type destination_uris: sequence of string
+        :param destination_uris: URIs of CloudStorage file(s) into which
+                                 table data is to be extracted.
+
+        :rtype: :class:`gcloud.bigquery.job.ExtractTableToStorageJob`
+        :returns: a new ``ExtractTableToStorageJob`` instance
```

Comment:
This is kind of redundant :smile:

How about, "The job to be executed." or something equally redundant.